### DISCUSSION TITLE
请问下这种情况怎么搞

### DISCUSSION THREAD

**Discussion by jefflee147** [gkd-kit/gkd#693] [Q&A]
我打开了授权  但是确显示用不了   索尼i5

|||
|-|-|
| ![Screenshot_20240807-172854.png](https://github.com/user-attachments/assets/a6003c56-974c-4a4b-b424-23f3004f22d1)|![Screenshot_20240807-172845.png](https://github.com/user-attachments/assets/76914f88-3a9f-4683-ad13-74aaff473b38)|

**Comment by lisonge** [comment]
应该是系统的问题，解决方式可以参考 #670

**Reply by jefflee147** [reply]
我卸载了 重新安装了一次就行了  谢谢
